cli/ballerina-cli/src/main/resources/command-info/add.toml

```diff
@@ -0,0 +1,9 @@
+[base_command]
+name = "add"
+short = "short description of add"
```

Comment:
Why don't we have the actual short description here?

Reply:
I will add actual descriptions for all toml file. Here I added sample descriptions for testing.